src/osd/osd_types.h

```diff
@@ -4618,6 +4618,9 @@ class pg_missing_set : public pg_missing_const_i {
   const std::map<hobject_t, item> &get_items() const override {
     return missing;
   }
+  std::map<hobject_t, item> &take_items() {
```

Comment:
take implies that missing is acutally removed, which appears here not to be the case.